src/utils/stacktrace.ts

```diff
@@ -0,0 +1,95 @@
+import ErrorStackParser from "error-stack-parser";
+
+export type RawStackFrames = string;
+
+const getErrorTitle = (e: Error): string => `${e.name || "Error"}: ${e.message}`;
+const getErrorStackFrames = (e: Error): string | undefined => e.stack?.replace(getErrorTitle(e) + "\n", "");
+
+export function captureRawStackFrames(filterFunc?: (...args: unknown[]) => unknown): RawStackFrames {
+    const savedStackTraceLimit = Error.stackTraceLimit;
+    const targetObj = {} as { stack: RawStackFrames };
+
+    Error.stackTraceLimit = 64;
```

Comment:
> Why are you using the value 64 here?

Because wdio could produce a lot of garbage frames, and 32 could possibly be not enough.

> Can we use some constant variable for it?

Yes, i will pull it out to somewhere